### DISCUSSION TITLE
So Long, and Thanks for all the Fish!

### DISCUSSION THREAD

**Discussion by Cito** [akosbalasko/yarle#161] [Show and tell]
Just want to give a big shout-out to @akosbalasko for providing this useful rope ladder! 🙏🙏🙏

Since I now have converted all of my ENEXes for good, I don't need Yarle any more, so you probably won't see any feature requests or  PRs by me any more. Thank you for the great support and discussions here.

Btw, this is how the graph of my ENEX notebooks converted to Obsidian looks like:

![My ENEXes converted to Obsidian](https://media.discordapp.net/attachments/709712341066842113/792883001234948096/my-evernote.png?width=728&height=742)

**Comment by akosbalasko** [comment]
Hi @Cito ! 

Wow, your graph looks great!
Thank you for all of your efforts, Yarle became better and better due to your improvements!
And who knows, maybe we can work together in the future on another project!

**Comment by gislepa** [comment]
I have downloaded Yarle from this site. When I try to run it, Windows Defender reports a security risk. Does someone else have the same experience?
